crates/plugins/mediator-coordination/src/web/mod.rs

```diff
@@ -10,15 +10,13 @@ use keystore::KeyStore;
 use std::sync::Arc;
 
 use crate::{
-    didcomm::bridge::{LocalDIDResolver, LocalSecretsResolver},
-    model::stateful::entity::{Connection, RoutedMessage, Secrets},
-    util,
+    didcomm::bridge::{LocalDIDResolver, LocalSecretsResolver}, model::stateful::entity::{Connection, RoutedMessage, Secrets}, util
 };
 
 pub fn routes(state: Arc<AppState>) -> Router {
     Router::new()
         // Unified route for all DIDComm messages
-        .route("/", post(handler::process_didcomm_message))
+        .route("/mediate", post(handler::process_didcomm_message))
```

Comment:
Why did you change the route of the didcomm requests?

Reply:
Just to seperate and make this clearer since the "/" route already exits though for a different method

Reply:
Are you aware that the mediator will interact with multiple clients? We need to ensure that all incoming requests are understood and handled correctly. Based on other implementations, mediation requests are typically directed to the "/" route.